src/App.jsx

```diff
@@ -1,5 +1,22 @@
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import ListContainer from './ListContainer';
+import RegionContainer from './RegionContainer';
+import { loadRegions, loadCategories } from './actions';
+import CategoriesContainer from './CategoriesContainer';
+
 export default function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
```

Comment:
한줄 띄주시면 가독성이  더 좋을거 같아요 :)
```suggestion
  const dispatch = useDispatch();
  
  useEffect(() => {
```